bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_madspin_card.dat

```diff
@@ -0,0 +1,31 @@
+#************************************************************
+#* MadSpin *
+#* *
+#* P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer *
+#* *
+#* Part of the MadGraph5_aMC@NLO Framework: *
+#* The MadGraph5_aMC@NLO Development Team - Find us at *
+#* https://server06.fynu.ucl.ac.be/projects/madgraph *
+#* *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+#initialization parameters
+#set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+set BW_cut 15 # cut on how far the particle can be off-shell
+#set max_weight_ps_point 400 # number of PS to estimate the maximum for each event
+#
+set max_running_process 1
+# specify the decay for the final state particles
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
In the proc, you already have l+ l- vl vl~ defined.
madspin and proc shares the particle definitions so remove the duplicated lines in either of the cards

Reply:
@Geet59 Please have a look at the questions related to WZ cards

Reply:
Hi @sihyunjeon I have removed the definitions from madspin card. Please have a look and let us know if it's fine now.